src/Carousel/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react'
+import Icon from '../Icon'
+import ButtonIcon from '../ButtonIcon'
+import cx from 'classnames'
+import {default as PT} from 'prop-types'
+
+const Carousel = ({ children, className }) => {
+  const [itemSelected, setItemSelected] = useState(0)
+
+  const moveNext = (inc = 1) => {
+    setItemSelected(itemSelected + inc)
+  }
+
+  const canMoveRight = itemSelected < children.length - 1
+  const canMoveLeft = itemSelected > 0
+
+  if (!children.length) return (
+    <div className={cx('ola_carousel', className)}>
+      <div className="ola_carousel-container">
+        {children}
+      </div>
+    </div>
+  )
+
+  return (
+    <div className={cx('ola_carousel', className)}>
+      <div className="ola_carousel-container">
+        <ButtonIcon
+          onClick={() => moveNext(-1)}
+          disabled={!canMoveLeft}
+          variant={canMoveLeft ? 'primary' : 'secondary'}
+        >
+          <Icon name="chevronLeft" size="big" />
+        </ButtonIcon>
+        {children.map((item, index) => {
+          return index === itemSelected ? React.cloneElement(item, {
+            key: index
+          }) : null
+        } )}
+        <ButtonIcon
+          onClick={() => moveNext()}
+          disabled={!canMoveRight}
+          variant={canMoveRight ? 'primary' : 'secondary'}
+        >
+          <Icon name="chevronRight" size="big" />
+        </ButtonIcon>
+      </div>
+      <div className="ola_carousel-dots">
+        {children.map((item, i) => (
+          <div
+            key={i}
+            onClick={() => setItemSelected(i)}
+            className={cx('ola_carousel-dots-item', i === itemSelected && 'selected')} />
```

Comment:
according with the other components, the state classname should start with `is-`. So this should be `is-selected`, instead `selected`.